app/models/timer.rb

```diff
@@ -0,0 +1,31 @@
+class Timer < ActiveRecord::Base
+
+  belongs_to :player
+  belongs_to :game
+
+  def initialize
+    @start_time = nil
+    @stop_time = nil
+    @running = false
+    @time_left = 500
+  end
+
+  def start
+    @time_left 
```

Comment:
Thanks - I remember setting this and have tried again with the new associations - 
```
def start
    @time_left = self.time_left
    @running = true 
    @start_time = Time.now.round(0)
  end
```